src/css/components/_live.scss

```diff
@@ -1,11 +1,22 @@
-.video-js.vjs-live .vjs-time-control,
-.video-js.vjs-live .vjs-time-divider,
-.video-js.vjs-live .vjs-progress-control {
-  display: none;
+.vjs-live-control {
+  display: block;
+  font-size: 1em;
+  line-height: 3em;
 }
 
-.video-js .vjs-live-control {
+.vjs-live .vjs-time-control,
+.vjs-live .vjs-time-divider,
+.vjs-live .vjs-progress-control {
```

Comment:
In keeping with the `vjs-hidden` stuff we were talking about, should we add the class to these in `updateShowing`?